### PR TITLE
Button States

### PR DESCRIPTION
**Pull request by wgranger**
This PR adds button down states to the app. I had to disable the press and hold functionality (that by default triggers a right-click). Now, buttons will change one color when pressed. The PR also defines a "tap" behavior, defined as touchdown + touchup. This is to keep a finger movement raising on a button from triggering that button.

I had some difficulties with inset drop shadows. WPF doesn't have this built-in, and there isn't a great looking workaround.

Simple buttons (Submit/Find another Galaxy) use the inset workaround, which you can see with the presence of an inner and outer border, but the answer buttons have a bit more complex rendering, and inset drop shadow makes the children of those button render badly.

I'll keep #34 open for now as a reminder to look for other possible solutions to inset drop shadows once more of the basic phase II functionality is complete.

### CODE REVIEW

**Comment by wgranger** [comment]
Made some updates here to get inset drop shadows working on answer borders. The solution here was to use a border element for each drop shadow needed (inset and normal dropshadow). 

Also, dropshadows tend to make text look fuzzy, so moving a dropshadow off of an element with text children, and onto it's own dropshadow-devoted element helps rendering here.

File: GalaxyZooTouchTable/GalaxyZooTouchTable/Behaviors/TapBehavior.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+        protected override void OnAttached()
+        {
+            base.OnAttached();
+
+            AssociatedObject.TouchDown += AssociatedObject_TouchDown;
+            AssociatedObject.TouchUp += AssociatedObject_TouchUp;
+        }
+
+        private void AssociatedObject_TouchUp(object sender, TouchEventArgs e)
```
[the remark this answers is not in the document]
Comment (wgranger) [reply GalaxyZooTouchTable/GalaxyZooTouchTable/Behaviors/TapBehavior.cs]: Of course. I should probably get back to using summary headers before classes and functions to define what is happening in the code.

**Comment by noahmalmed** [comment]
conflicts

**Comment by wgranger** [comment]
Fixed

**Comment by noahmalmed** [comment]
A note from testing:

 When there is no galaxy selected, you can still tap the smooth/features buttons. Meaning you can still see the touch feedback, but nothing happens after you touch them. Maybe the buttons should be disabled tapped.

Otherwise the UI update looks really good!

**Comment by wgranger** [comment]
I can take care of this pretty quickly.

**Comment by wgranger** [comment]
Should be good to go 👍

**Pull request merged**